src/database/migrations/0067-add-funder-collaborative-members.sql

```diff
@@ -0,0 +1,77 @@
+ALTER TABLE funders ADD COLUMN is_collaborative boolean NOT NULL DEFAULT false;
+CREATE TABLE funder_collaborative_members (
+	funder_collaborative_short_code short_code_t NOT NULL
+	REFERENCES funders (short_code) ON DELETE CASCADE,
+	member_funder_short_code short_code_t NOT NULL
+	REFERENCES funders (short_code) ON DELETE CASCADE,
+	-- If the user that created this happens to be deleted,
+	-- we still don't want to delete this.
+	created_by uuid REFERENCES users (keycloak_user_id)
+	ON DELETE SET NULL,
+	created_at timestamp with time zone NOT NULL DEFAULT now(),
+	not_after timestamp with time zone DEFAULT null,
+	PRIMARY KEY (funder_collaborative_short_code, member_funder_short_code),
+	CONSTRAINT no_self_membership CHECK (
+		funder_collaborative_short_code != member_funder_short_code
+	)
+
+);
+
```

Comment:
We (really I) forgot a `SELECT audit_table('funder_collaborative_members');` here.